src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.scss';
-import App from './components/container/App/App';
+import App from './components/components/App/App';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
```

Comment:
please revert this file to the original state (from main branch), as it would lead to merge conflicts otherwise